### ISSUE TITLE
Checkboxes in banner set to visually-hidden

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_eu_cookie_compliance#1]
When using the banner, there are checkboxes set to visually-hidden. Users shouldn't interact with these, but they capture keyboard focus (meaning e.g. two extra tabs to reach the skip link) and might also disrupt things for screenreader users.

Accessibility checkers flag this as a failure against [Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) as there is no label - but they should just be hidden.

It might be more suitable to set this element to `hidden` rather than `visually-hidden`.

https://github.com/localgovdrupal/localgov_eu_cookie_compliance/blob/29226a228caf427b004aa75e88ce847e607f531c/templates/eu-cookie-compliance-popup-info--override.html.twig#L84

**Comment by andybroomfield** [comment]
Also encountering this.
Ref comment 
> {# This hidden markup is needed by Drupal.eu_cookie_compliance.setPreferenceCheckboxes() #}

Does it need to be visually hidden, or can it be hidden.
Changing to hidden does appear to fix the accessbility tabbing issue, and it does allow the cookies to be removed / set.

**Comment by keelanfh** [comment]
Resolved by #5
